roles/kubernetes-apps/ansible/templates/netchecker-server-deployment.yml.j2

```diff
@@ -23,6 +23,11 @@ spec:
             requests:
               cpu: {{ netchecker_server_cpu_requests }}
               memory: {{ netchecker_server_memory_requests }}
+{% if podsecuritypolicy_enabled %}
```

Comment:
can't we just add the securityContext in all cases?

Reply:
@ant31 I didn't want to change the default behaviour but why not

Reply:
Fixed in afca263